ibc-core/ics03-connection/types/src/error.rs

```diff
@@ -9,11 +9,11 @@ use ibc_primitives::{Timestamp, TimestampError};
 
 #[derive(Debug, Display)]
 pub enum ConnectionError {
-    /// client error: `{0}`
+    /// client error: {0}
     Client(ClientError),
-    /// decoding error: `{0}`
+    /// decoding error: {0}
     Decoding(DecodingError),
-    /// host error: `{0}`
+    /// host error: {0}
```

Comment:
Backticks (`) are dropped here. We are having it in bunch of others. I wonder if it was intentional?

Reply:
This is explained in the [ADR](https://github.com/cosmos/ibc-rs/pull/1344)